draft-ietf-quic-recovery.md

```diff
@@ -537,8 +537,15 @@ and otherwise it MUST send an Initial packet in a UDP datagram of at least
 1200 bytes.
 
 The crypto retransmission timer is not set if the time threshold
-{{time-threshold}} loss detection timer is set.  When the crypto
-retransmission timer is active, the probe timer ({{pto}}) is not active.
+{{time-threshold}} loss detection timer is set.  The time threshold loss
+detection timer is expected to both expire earlier than the crypto
+retransmission timeout and be less likely to spuriously retransmit data.
+The Initial and Handshake packet number spaces typically have a small number
+of packets in them, so time threshold loss detection will typically declare
+packets lost before packet threshold.
```

Comment:
I think what you want to say is that time-threshold is more likely to be used for loss detection than packet-threshold, since there just might not be enough packets. So how about just this, given that you've just talked about time-threshold: "The Initial and Handshake packet number spaces will typically carry a small number of packets, so losses are less likely to be detected using packet-threshold detection."

That said, I still am not sure that it's worth mentioning the packet-threshold here at all.  But my opinion is not strong, so your call.

Reply:
Thanks, suggestion taken.  I think why you'd arm the loss-detection timer is non-obvious, so I thought more explanatory text was better than less, and I think the small number of packets argument is relevant.